srcpkgs/armcord/template

```diff
@@ -0,0 +1,55 @@
+
+# Template file for 'armcord'
+pkgname=armcord
+version=3.1.4
+revision=1
+archs="x86_64* i686* aarch64* armv7l*"
+hostmakedepends="nodejs electron19 git"
```

Comment:
optimally, electron21+ should be used if available, for best support

Reply:
We don't have electron21 unfortunately